demos-europe/demosplan-core
# feat(refs T29570): integration masterportal

**Ticket:** https://yaits.demos-deutschland.de/T29570

We need to use masterportal in any way and decided to use the masterportalApi. We only create the map with the Api and then add layers and everything else with our own components the way it worked before. This is done in DpOlMap.

Additionally, the baselayer and layerLayers were replaced in all projects, see https://demosdeutschland.slack.com/archives/C03R33ANPRB/p1668598402623519 for more information.

### How to review/test
Check various maps (where DpOlMap is used) and see if you can zoom, move and do all the fun stuff with maps as you could before. EG Verfahren -> Planungsdokumente und Planzeichnung

Info: I couldn't test all the maps, because some things do not work atm. Needs to be tested again when main works again. 

### Linked PRs (optional)
https://github.com/demos-europe/demosplan-project-robobsh/pull/6
https://github.com/demos-europe/demosplan-project-bimschgsh/pull/7
https://github.com/demos-europe/demosplan-project-bobhh/pull/8
https://github.com/demos-europe/demosplan-project-bobsh/pull/5
https://github.com/demos-europe/demosplan-project-diplanbau/pull/25
